lambdas/totp-generator/src/totp-generator-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { APIGatewayProxyResult } from "aws-lambda";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+
+import TOTP from "totp-generator";
+import { Logger } from "@aws-lambda-powertools/logger";
+const TOTP_TTL_IN_SECONDS = 30;
```

Comment:
Thought: We should make the TTL a parameter instead of hard-coding it so it's more configurable.

Reply:
I suppose we could, but i think we can defer this till it becomes evident it's needed